### PR TITLE
improve german translation

### PR DESCRIPTION
**Pull request by larsgrefer**
@StefMa Please take a look on this

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -185,6 +187,7 @@
     <string name="none">Keine(r)</string>
     <string name="section_issue_labels">Markierungen:</string>
     <string name="log_in">Einloggen</string>
+    <string name="log_out">Ausloggen</string>
```
Comment (StefMa) [inline app/src/main/res/values-de/strings.xml]: The 'rightest' (omg xD) would be 'Abmelden'. But then we need 'Anmelden' for log_in to..

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -70,6 +70,7 @@
     <string name="loading_commits">Lade Commits…</string>
     <string name="loading_files_and_comments">Lade Dateien &amp; Kommentare…</string>
     <string name="loading_refs">Lade Branches &amp; Tags…</string>
+    <string name="loading">Laden…</string>
```
Comment (StefMa) [inline app/src/main/res/values-de/strings.xml]: Should be 'Lade...' without n

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (larsgrefer) [inline app/src/main/res/values-de/strings.xml]: But in the next line 'anmelden' is used for 'sign up'

Comment (StefMa) [inline app/src/main/res/values-de/strings.xml]: Than it is right. 
But normaly "signup" is equal to "Registrieren" and "log_in" "Anmelden".
But we have signup always translated as "sign up" also "Anmelden". We shouldn't break that.

So Ausloggen is ok ;)

**Comment by StefMa** [comment]
Ok, have made a full review. It's all ok and can be merged from a german user ;)

**Comment by larsgrefer** [comment]
As far as I know we don't have a German user with write-access to this repository, have we?

**Comment by hzsweers** [comment]
No, we don't. It also doesn't have to be merged by a German user. We've made our stance clear about not adding more collaborators right now. These passive naggy comments are annoying and don't make us inclined to reconsider, so please stop.

**Pull request merged**